document/README.md

```diff
@@ -21,3 +21,65 @@ To build everything and update [webassembly.github.io/spec](https://webassembly.
 make publish
 ```
 Please make sure to only use that once a change has approval.
+
+## Step by step guide to building the spec
+
+Make sure this repository was cloned with `--recursive`:
+
+```
+git clone --recursive https://github.com/WebAssembly/spec
+```
+
+If you have already cloned but without `--recursive`, you can delete and re-clone, or `cd` into `spec` and run:
+
+```
+git submodule update --init --recursive
+```
+
+The rest of these instructions assume you are in the directory where is README is:
+
+```
+cd spec/document
+```
+
+You will need `python3.7`, and `pip`. `pip` should come with Python, if not follow [these installation instructions for `pip`](https://pip.pypa.io/en/stable/installing/), or check your system package manager for `pip3`.
+
+> Important: you will need the version of pip that works with `python3.7`.
+
+
+Use something like [`pipenv`](https://pipenv.pypa.io/) to keep your system installation of Python clean.
+
+```
+pip install pipenv
+pipenv --python 3.7
+pipenv shell
+```
+
+Install Python dependencies:
+
+```
+pip install Sphinx==2.4.4
+```
+
+You can now build the [multi-page html document](https://webassembly.github.io/spec/core/):
+
+```
+make -C core html
+```
```

Comment:
Would be good to also include the PDF target. Sphinx and MathJax are rather sloppy and forgiving, so that most errors in LaTeX bits of the doc only surface when you build the PDF. Moreover, the PDF is included in the deployment, so the build needs to work.